### PR TITLE
feat: bootstrap the operator

### PR DESCRIPTION
**Pull request by xcoulon**
result of the `operator-sdk new` command
also: update the `README` (converted to asciidoc)
also: update the .gitignore to exclude all IDEs/OSes
files

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+= toolchain-operator
+
+This is the CodeReady Toolchain Operator repository
```
Comment (dipak-pawar) [inline README.adoc]: maybe we should change this to member operator

Comment (xcoulon) [reply README.adoc]: did you mean `host operator`?

Comment (dipak-pawar) [reply README.adoc]: yes

Comment (xcoulon) [reply README.adoc]: @dipak-pawar see the change in https://github.com/codeready-toolchain/host-operator/pull/2/commits/512ab01502cb0b66115ce341552fd0dc873cf283

**Pull request merged**